Jenkinsfile

```diff
@@ -2,7 +2,7 @@
 //Leave the above line alone.  It identifies this as a groovy script.
 @Library('vs-common-build') _
 
-def lvVersions = ['2017']
+def lvVersions = ['2017','2016']
```

Comment:
The ni-veristand-cds versions of the Classes and FXP repos do not have 2016 listed as a supported version. This would cause the build to fail without those being updated.